packages/beacon-node/src/network/networkConfig.ts

```diff
@@ -1,6 +1,6 @@
 import {BeaconConfig} from "@lodestar/config";
-import {NodeId, computeNodeId} from "./subnets";
-import {CustodyConfig, computeCustodyConfig} from "../util/dataColumns";
```

Comment:
I am surprised our linter doesn't catch this

Reply:
we do have https://biomejs.dev/linter/rules/use-import-extensions/ though, not sure how this didn't catch the case here

Reply:
yes we should have catch this during our linting process, otherwise we'll have similar issues in the future

Reply:
it's seems the linter catches this though? I haven't tested the specific case but did a quick sanity check on unstable and it's complains about missing file extension there

Reply:
They're caught, but lints/tests aren't passing on the root `peerDAS` branch right now, so not all the errors are fixed there.